tests/test.ts

```diff
@@ -163,21 +165,41 @@ describe.skip('Test clasp version and versions function', () => {
   });
 });
 
-describe.skip('Test getScriptURL function from utils', () => {
+describe('Test getScriptURL function from utils', () => {
   it('should return the scriptURL correctly', () => {
     const url = getScriptURL('abcdefghijklmnopqrstuvwxyz');
     expect(url).to.equal('https://script.google.com/d/abcdefghijklmnopqrstuvwxyz/edit');
   });
 });
 
-describe.skip('Test getFileType function from utils', () => {
+describe('Test getFileType function from utils', () => {
   it('should return the lowercase file type correctly', () => {
     expect(getFileType('SERVER_JS')).to.equal('js');
     expect(getFileType('GS')).to.equal('gs');
     expect(getFileType('JS')).to.equal('js');
   });
 });
 
+describe('Test getAPIFileType function from utils', () => {
+  it('should return the uppercase file type correctly', () => {
+    expect(getAPIFileType('file.GS')).to.equal('SERVER_JS');
+    expect(getAPIFileType('file.JS')).to.equal('SERVER_JS');
+    expect(getAPIFileType('file.txt')).to.equal('TXT');
+  });
+});
+
+// NOTE: we should make saveProjectId async because dotf.write is async
+describe('Test saveProjectId function from utils', () => {
+  it('should save the scriptId correctly', () => {
+    spawnSync('rm', ['.clasp.json']);
+    saveProjectId('12345');
+    setTimeout(() => {
```

Comment:
This is here because otherwise, it won't have saved the projectId before running the test.

Reply:
Yeah I was just putting that there so the tests didn't fail. I'll go ahead and make another commit to this branch with an update of `saveProjectId`